tests/unit/loader/loader-test.js

```diff
@@ -0,0 +1,208 @@
+QUnit.module("igLoader Common", {
+	errors: [],
+	jsPath: "base/mock/src/js/",
+	cssPath: "base/mock/src/css/",
+
+	beforeEach: function(){
+		this.cleanScripts();
+	},
+
+	resourcesLoaded: function(expectedResources, assert) {
+		//check if expected resources are loaded
+		var scripts = $(document).find("head > script"), actualPaths = [];
+		scripts.each(function () {
+			//console.log("script.src: " + this.src)
+			if (this.src.indexOf("ignite-ui/src/js/") || this.src.indexOf("igniteui/src/js/") > 0) {
```

Comment:
`this.src.indexOf("ignite-ui/src/js/")` doesn't really check anything (kind of why those actually work), so maybe it should use the `jsPath`? Same for the css.